scripts/update-hackage.nix

```diff
@@ -20,6 +20,10 @@ import ./update-external.nix args {
 
     hackage-to-nix hackage.nix
 
+    echo "Running update-index-state-hashes..."
+
+    ${update-index-state-hashes}/bin/update-index-state-hashes > ./index-state-hashes.nix
```

Comment:
Should this be called *after* cd-ing into `hackage.nix`? As it is it looks like it won't modify the copy in `hackage.nix`. And indeed the file hasn't been modified in `hackage.nix` in some time.